guides/10_Video_Processor.md

```diff
@@ -0,0 +1,115 @@
+# Video Processing APIs
+
+## Introduction
+
+Amazon Chime SDK for JavaScript contains APIs to facilitate the processing of video stream and provides a simplified workflow for adding processing to the outgoing video feed.
```

Comment:
Nit: Sorta redundant and run on.  Probably can just be `...contains easy to use APIs for adding frame-by-frame processing to an outgoing video stream.` More concise and IMO more explicit.  'Simplified' is an imprecise word here because its not clear what its a simplified version of :P .